### PR TITLE
chore: crypto-ffi: remove autogenerated Swift source

### PR DESCRIPTION
**Pull request by istankovic**
We don't really need it.

----                                                                                            
##### PR Submission Checklist for internal contributors                                         
                                                                                                
- The **PR Title**                                                                              
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`                                 
  - [x] answers the question: _If merged, this PR will: ..._ ³                                  
                                                                                                
1. https://sparkbox.com/foundry/semantic_commit_messages                                        
1. https://github.com/wireapp/.github#usage                                                     
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by augustocdias** [comment]
Could you use the template for the PR description body? It is not required to fill everything but at least the checklist and the description of what's done

File: crypto-ffi/build.rs
Comment (augustocdias) [inline crypto-ffi/build.rs:1]: I think these changes are from your other PR that you're investigating right?

**Comment by augustocdias** [comment]
I am not sure if git will keep the empty folders, but in case not, you can delete the whole `Sources` folder.

File: crypto-ffi/build.rs
Comment (istankovic) [reply crypto-ffi/build.rs:1]: Ah, yes, that got into this by accident, will remove. :+1:

**Comment by istankovic** [comment]
> I am not sure if git will keep the empty folders, but in case not, you can delete the whole `Sources` folder.

Yeah, git doesn't track directories so once we remove all files, those will be gone too.

**Comment by istankovic** [comment]
@augustocdias added the checklist, however, one checkbox cannot be checked since there is no JIRA item associated with this.

**Review by augustocdias — APPROVED** [review]

**Pull request merged**